examples/GettingStarted/BinaryClassification_SentimentAnalysis/BinaryClassification_SentimentAnalysis.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\datasets\imdb_labeled.txt" Link="datasets\imdb_labeled.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="..\..\datasets\yelp_labeled.txt" Link="datasets\yelp_labeled.txt">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="0.1.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="datasets\" />
+  </ItemGroup>
+
+</Project>
```

Comment:
What is the rationale behind creating a separate project for "Binary Classification sentiment analysis"?

Reply:
We want to have separate project for different problem types so it aligns with our docs